jsplot/src/main/java/tech/tablesaw/plotly/Plot.java

```diff
@@ -28,8 +31,8 @@ public static void show(Figure figure, String divName, File outputFile) {
         String output = page.asJavascript();
 
         try {
-            try (FileWriter fileWriter = new FileWriter(outputFile)) {
-                fileWriter.write(output);
+            try (Writer writer = new OutputStreamWriter(new FileOutputStream(outputFile), StandardCharsets.UTF_8)) {
```

Comment:
why are you changing this?

Reply:
FileWriter uses ```file.encoding``` encoding and if there is no ```file.encoding``` encoding, Java uses UTF-8.
Sometimes ```file.encoding``` is not UTF-8 and I forced to use UTF-8.

Reply:
Won't this still use UTF-8 encoding since it's passing `StandardCharsets.UTF_8`?

Reply:
Oh, I misunderstood you comment. You want to force it to use UTF-8. That makes sense. Thanks

Reply:
Yes. I want to force it to use UTF-8.